Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -66,6 +66,11 @@ public struct _NameMap: ExpressibleByDictionaryLiteral {
     /// other.
     case unique(proto: StaticString, json: StaticString)
 
+    /// Used for enum cases only to represent a value's primary name (the first
+    /// defined case) and its aliases. The JSON and text format names for enums
+    /// are always the same.
+    case aliased(primary: StaticString, aliases: [StaticString])
```

Comment:
Should this be aliased(proto:aliases:) to follow the naming of the other?  i.e. - it is a refinement of same where there are aliases?